README.md

```diff
@@ -103,6 +103,14 @@ or, for newer versions of Gradle:
 ```
 purgeSnapshots=1 ./gradlew test
 ```
+
+## Failing on Missing Snapshots
+In order to fail if a snapshot does not exist, you just need to execute your tests like this:
+
+```
+failOnMissingSnapshots=1 ./gradlew test
```

Comment:
Can we just assign it to true or false instead of using numeric values? I know the old API specifies other values like: updateSnapshots but it's time to make a better use of the config params I think :) Maybe the best solution to avoid breaking changes we can use both config values to enable this option.

Reply:
I will have a look into it. Oriented myself on the other config parameters, but this should be possible.

Reply:
We keep using these env vars because the old fork created them. But I think it's time to at least improve this scenario by using boolean params. Remember we should also be able to use "1" as env var and maintain the original behavior if we don't want to introduce any breaking change 😃

Reply:
Absolutely :-) – Will have a look if this is possible with a small change – Otherwise, we might want to do this in a separate PR, if that is okay for you.

Reply:
ok!